00 Intro/00 BoilerPlate/README.md

```diff
@@ -4,10 +4,10 @@ In this sample we are going to setup a web project that can be easily managed
 by webpack.
 
 We will setup an initial npm project, give support to ES6, and install webpack.
-
-We will create a hellword.js sample.
+Then we will create a hellword.js sample.
```

Comment:
Typo: helword, should it be helloworld.js? (They typo was as well on the original versión)